README.md

```diff
@@ -309,8 +309,8 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Aniket Prabhakar](https://github.com/aniketprabhakar)
   - [Aniket Singh](https://github.com/aniketsingh98571)
   - [Animesh Pathak](https://github.com/sonichigo)
-  - [Anirudh](https://github.com/aspiringDev22)
   - [Aniruddha Inge](https://github.com/Aniruddha-Inge)
+  -  [Anirudh](https://github.com/aspiringDev22)
```

Comment:
```suggestion
  - [Anirudh](https://github.com/aspiringDev22)
```
Please remove the space.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix